encrypted-media-respec.html

```diff
@@ -370,30 +401,29 @@
-              (The indelible value(s) MUST NOT ever be provided to the application, even in opaque or encrypted form.) 
-            </p> 
+              (The <a def-id="distinctive-permanent-identifier-maybe-plural"></a> MUST NOT ever be provided to the application, even in opaque or encrypted form.) 
+            </p>
           </div>
-          <p class="issue"><a href="https://github.com/w3c/encrypted-media/issues/117">Issue 117</a> - Ensure providing "permanent" identifiers, such as to an <a href="#privacy-individualization">individualization</a> server normatively triggers conditions related to distinctive identifiers.</p>
-          <p>A distinctive identifier is a piece of data, implication of the possession of a piece of data, or an observable behavior or timing for which all of the following criteria hold:</p>
+          <p>A distinctive identifier is a value, piece of data, implication of the possession of a piece of data, or an observable behavior or timing for which all of the following criteria hold:</p>
           <ul>
-            <li><p>It is not shared across a large population of users or client devices.</p></li>
+            <li><p>It is <em>not</em> shared across a large population of users or client devices.</p></li>
             <li>
-              <p>It, information about it, or values derived or otherwise related to it are exposed, even in encrypted form, outside the client.
+              <p>It, information about it, or values derived from or otherwise related to it are exposed, even in encrypted form, outside the client.
                 This includes but is not limited to providing it to the application and/or license, <a href="#privacy-individualization">individualization</a>, or other server.
               </p>
             </li>
             <li><p>It has one or more the following properties:</p>
               <ul>
-                <li><p>It is derived from a value that is indelible in some way, including but not limited to hardware, operating system installation instance, user agent installation instance, <a def-id="cdm"></a> or other software component, confguration file or similar semi-permanent data, or client or other user account values.</p></li>
-                <li><p>The generation, <a href="#privacy-individualization">individualization</a>, provisioning or other process that produced the value involved, used, provided, derived from, or similarly involved such an indelible value or another distinctive identifier.</p></li>
+                <li><p>It is derived from one or more <a def-id="distinctive-permanent-identifier-maybe-plural"></a>.</p></li>
+                <li><p>The generation, <a href="#privacy-individualization">individualization</a>, provisioning or other process that produced the value involved, used, provided, derived from, or similarly involved one or more <a def-id="distinctive-permanent-identifier-maybe-plural"></a> or another distinctive identifier.</p></li>
```

Comment:
Pedantically (and not blocking us progressing), an example of a "process that produced a value" could be simply copying. So a copy of a Distinctive Permanent Identifier is a Distinctive Identifier (by the above definition) and since it has the same value as the Distinctive Permanent Identifier the Distinctive Permanent Identifier is also a Distinctive Identifier.